README.md

```diff
@@ -28,6 +28,233 @@
 
 # Triton Inference Server
 
-> **Note** <br>
-> You are currently on the r23.03 branch which tracks stabilization 
-> towards the next release.<br> This branch is not usable during stabilization.
\ No newline at end of file
+[![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+**LATEST RELEASE: You are currently on the main branch which tracks
```

Comment:
Is this message correct? We are not on the `main` branch since this PR is going to merge in `r23.03` branch.

Reply:
comment has been addressed